app/enterprise/1.5.x/kong-for-kubernetes/install-on-kubernetes.md

```diff
@@ -0,0 +1,310 @@
+    ```
+3. Create secret:
+    ```
+    $ oc create secret generic kong-session-config -n kong --from-file=admin_gui_session_conf=<manager-config-filename> --from-file=portal_session_conf=<portal-config-filename>
+    ```
+{% endnavtab %}
+{% endnavtabs %}
+## Step 7. Prepare Kong's configuration file
+
+1. Create a `values.yaml` file based on the sample in the [Kong charts repository](https://github.com/Kong/charts/blob/master/charts/kong/values.yaml). This file sets all the necessary parameters for your Kong environment.
+
+2. Minimally, for setting up Kong Enterprise on Kubernetes, you will need to set the following parameters:
+
+    |Parameter      | Value                         |
+    |---------------|-------------------------------|
+    |`env.database` | `"postgres"` or `"cassandra"` |
+    |`env.password.valueFrom.secretKeyRef.name` | Name of secret that holds the super admin password. In the example above, this is set to `kong-enterprise-superuser-password`. |
+    |`env.password.valueFrom.secretKeyRef.key` | The type of secret key used for authentication. If you followed the default settings in the example above, this is `password`. |
+    |`image.repository` | The Docker repository. In this case, `kong-docker-kong-enterprise-edition-docker.bintray.io/ kong-enterprise-edition`. |
+    |`image.tag` | The Docker image tag you want to pull down, e.g. `"1.5.0.2-alpine"`. |
+    |`ingressController.enabled` | Set to `true` if you want to use the Kong Ingress Controller, or `false` if you don't want to install it. |
+
+3. In the `Kong Enterprise` section, enable Kong Manager (`manager`) and Kong Dev Portal (`portal`).
+
+    For example:
+    ```
+    manager:
+      enabled: true
+      type: LoadBalancer
+      http:
+        enabled: true
+        servicePort: 8002
+      tls:
+        enabled: true
+        servicePort: 8445
+
+    portal:
+      enabled: true
+      type: LoadBalancer
+      http:
+        enabled: true
+        servicePort: 8003
+      tls:
+        enabled: true
+        servicePort: 8446
+    ```
+
+4. Fill in the rest of the parameters as they fit your implementation. Use the comments in the sample file to guide you, and see the documentation on [Kong Enterprise parameters](https://github.com/Kong/charts/blob/master/charts/kong/README.md#kong-enterprise-parameters) for more details.
+
+## Step 8. Deploy Kong Enterprise on Kubernetes
+The steps in this section show you how to install Kong Enterprise on Kubernetes using Helm.
+>Note: the following instructions assume that you're running Helm 3.
+
+{% navtabs %}
+{% navtab kubectl %}
+1. Run:
+    ```
+    $ helm install  my-kong kong/kong --namespace kong --values ~/values.yaml
+    ```
+    This may take some time.
+
+2. Check pod status:
+    ```
+    $ kubectl get pods -n kong
+    ```
+    Once migrations are complete and the `my-kong-kong-<ID>` pod is running, move on to the next section.
```

Comment:
Prefer "After"  to "Once" to avoid ambiguity. This is a former employers style guide thing. Global comment.